File: cliente-desktop/Código/PFC_V1/Modelo/Hidrometro.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using LiteDB;
using PFC_V1.Modelo;

namespace PFC_V1
{
    public class Hidrometro : Objeto
    {
		public string identificador { get; set; }
		public string chave { get; set; }
		public string modelo { get; set; }
		public string descricao { get; set; }
		public bool ativo { get; set; }
		public List<Registro> registros { get; set; }
		public List<Alerta> alertas { get; set; }
		public List<Regra> regras { get; set; }
	}
}

File: cliente-web/src/app/_models/Hidrometro.ts
import { Objeto } from './Objeto';
import { Registro } from './Registro';
import { Alerta } from './Alerta';
import { Regra } from './Regra';

export class Hidrometro {

    id: number;
    identificador: string;
    chave: string;
    modelo: string;
    descricao: string;
    ativo: boolean;
    registros: Registro[];
    alertas: Alerta[];
    regras: Regra[];
}
